src/migrations/hotbar-store/5.0.0-beta.4.ts

```diff
@@ -0,0 +1,30 @@
+import type { Hotbar } from "../../common/hotbar-store";
+import { migration } from "../migration-wrapper";
+import { catalogEntityRegistry } from "../../renderer/api/catalog-entity-registry";
+
+export default migration({
+  version: "5.0.0-beta.4",
```

Comment:
I think this needs to be bumped to `beta.5` because `beta.4` has already been released.